database/migration/aruha-522-timelines.sql

```diff
@@ -0,0 +1,23 @@
+SET ROLE zalando_nakadi_data_owner;
+
+CREATE TABLE zn_data.storage (
+  st_id            VARCHAR(36) NOT NULL PRIMARY KEY,
+  st_type          VARCHAR(32) NOT NULL,
+  st_configuration JSONB       NOT NULL
+);
+
+
```

Comment:
@bocytko @valgog 
- Ticket passed thru review of all the team members, so I believe that team adopted. 
- We do not use any automation now that relies on the rule you are saying about.
So I think that conversation should be closed, because it stops us from releasing this ticket.